src/unstract/sdk/llm.py

```diff
@@ -22,7 +22,7 @@
 class ToolLLM:
     """Class to handle LLMs for Unstract Tools."""
 
-    code_block_regex = re.compile(r"```.*?\n(.*?)\n```", re.DOTALL)
+    json_regex = re.compile(r"\{(?:.|\n)*\}")
```

Comment:
Can you update the PR description with why this change was needed?

Reply:
If I understand it right, we are ignore all JSONs returned by the LLM calls right? Do review this, but this might imply a regression for example in the context of workflow generation with a prompt (I know we barely use this feature but still)